packer/builder/azure/arm/tempname.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Open Technologies, Inc.
+// All Rights Reserved.
+// Licensed under the Apache License, Version 2.0.
+// See License.txt in the project root for license information.
+
+package arm
+
+import (
+	"fmt"
+
+	"github.com/Azure/packer-azure/packer/builder/azure/common/utils"
+)
+
+const (
+	ResourceGroupNameTemplate = "packer-Resource-Group-%s"
+)
+
+type TempName struct {
+	alphabet string
+	suffix   string
+
+	ComputeName       string
+	DeploymentName    string
+	ResourceGroupName string
+	OSDiskName        string
+}
+
+func NewTempName() *TempName {
+	tempName := &TempName{
+		alphabet: "0123456789bcdfghjklmnpqrstvwxyz",
```

Comment:
hmmm, why save the alphabet in every struct? might as well just have it as a const?